react/features/base/dialog/components/native/styles.js

```diff
@@ -12,6 +12,47 @@ const DIALOG_BORDER_COLOR = 'rgba(255, 255, 255, 0.2)';
 export const FIELD_UNDERLINE = ColorPalette.transparent;
 export const PLACEHOLDER_COLOR = ColorPalette.lightGrey;
 
+/**
+ * Default styles for the items of a {@code BottomSheet}-based menu.
+ *
+ * These have been implemented as per the Material Design guidelines:
+ * {@link https://material.io/guidelines/components/bottom-sheets.html}.
+ */
+const bottomSheetItemStyles = createStyleSheet({
+    /**
+     * Container style for a generic item rendered in the menu.
+     */
+    style: {
```

Comment:
Is it common to have a style called style?

Reply:
Not really, but that is a special style object automagically used by the AbstractButton component, and that expects it to be ```style, iconStyle, labelStyle, underlayColor```